src/nvim/popupmnu.c

```diff
@@ -201,9 +201,15 @@ void pum_display(pumitem_T *array, int size, int selected, bool array_changed)
     }
 
     // If there is a preview window above, avoid drawing over it.
+    // Do keep at least 10 entries.
     if (pvwin != NULL && pum_row < above_row && pum_height > above_row) {
```

Comment:
in the vim patch the condition is:

    if (pvwin != NULL && pum_row < above_row && pum_height > 10)

did you intentionally change it?

Reply:
I did not, that's a typo. `pum_height > 10` is the correct condition in this version.